tools/run-image-exit-codes.toit

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2024 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
```

Comment:
Maybe add a comment here that you can't just change these, because we may have already deployed boot.sh scripts and they will all break.

Reply:
done.